pingrid/impl.py

```diff
@@ -90,16 +81,234 @@ def error_fig(error_msg="error"):
 FuncInterp2d = Callable[[Iterable[np.ndarray]], np.ndarray]
 
 
-class BGRA(NamedTuple):
-    blue: int
-    green: int
+class ColorScale:
+    """A class to define and manipualte colorscales.
+
+    `ColorScale` instances are defined by colors at anchor points and are named.
+    Colors are graduated from one anchor to the next.
+    To make a band of the same color, following anchors should be equal.
+    To mark a discontinuity, following colors should differ and have the same anchor value.
```

Comment:
I would be nice then to print an image of the Examples colorscale. More generally speaking, it would be nice to print an image of all our registered colorscales. Can we do that?